tests/Microsoft.VisualStudio.ProjectSystem.Managed.UnitTests/ProjectSystem/Debug/DebugTokenReplacerTests.cs

```diff
@@ -68,12 +68,14 @@ public async Task ReplaceTokensInProfileTests()
     [InlineData("this is msbuild: %env3% $(msbuildProperty2) $(msbuildProperty3)",  "this is msbuild: Property6 Property2 Property3", true)]
     [InlineData(null, null, true)]
     [InlineData(" ", " ", true)]
-    public async Task ReplaceTokensInStringTests(string input, string expected, bool expandEnvVars)
+    public async Task ReplaceTokensInStringTests(string? input, string? expected, bool expandEnvVars)
     {
         var replacer = CreateInstance();
 
         // Test msbuild vars
-        string result = await replacer.ReplaceTokensInStringAsync(input, expandEnvVars);
+        
+        // TODO: fix nullable annotations: https://github.com/dotnet/project-system/issues/9762
```

Comment:
As commented on that issue, I don't think we'll change the annotation.

This can just be a comment explaining that although the API doesn't advertise that it accepts null, it does handle it and we test that here to preserve back compat.